tests/LINEBot/Util/MockUtil.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+namespace LINE\Tests\LINEBot\Util;
+
+use PHPUnit\Framework\TestCase;
+
+class MockUtil
+{
+    public static function builder(TestCase $testRunner, $class, $postfix, $method = 'build')
```

Comment:
Mocking builder method like this will not run actual code. It doesn't make sense to me.
All builder classes are test friendly. Create a instance on the test code and make sure the bult json as string.
Plus we can show usage to users how to use the builder class.